nav_admin.php

```diff
@@ -2,7 +2,7 @@
 <nav>
 	<ul>
 		<li><a href = "#">Setting</a>
-			<ul>
+			<ul id = "a">
```

Comment:
This is make some change to that it can defined the difference between the drop down menu for setting with drop down menu for the table(query)